.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,8 @@
 - [ ] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/.github/CONTRIBUTING.md)
 - [ ] [Sign Gradle CLA](http://gradle.org/contributor-license-agreement/)
 - [ ] [Link to Design Spec](https://github.com/gradle/gradle/tree/master/design-docs) for changes that affect more than 1 public API (that is, not in an `internal` package) or updates to > 20 files
-- [ ] Integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective. Unit tests (under `<subproject>/src/test`) to verify logic
+- [ ] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective. 
```

Comment:
Minor nitpick. This sentence has a dot at the end whereas other sentences don't have it. We should standardize.